service/src/main/resources/application.yml

```
@@ -57,6 +57,15 @@ drshub:
         - type: https
           auth: current_request
           fetchAccessUrl: true # Used for Azure
+    sage:
+      name: Sage Bionetworks
+      hostRegex: 'repo-([^.]+)\.\1\.sagebase\.org' # Dev: 'repo-dev.dev.sagebase.org' & Prod: 'repo-prod.prod.sagebase.org'
```

no need to be so fancy
```suggestion
      hostRegex: 'repo-dev\.dev\.sagebase\.org|repo-prod\.prod\.sagebase\.org'
```